AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -793,9 +770,25 @@ - (void)layoutSubviews
       [self performBatchAnimated:YES updates:^{
         [_dataController relayoutAllNodes];
       } completion:nil];
+      // We need to ensure the size requery is done before we update our layout.
+      [self waitUntilAllUpdatesAreCommitted];
```

Comment:
Haven't tested rotation but this was necessary to support an image carousel that resizes while it scrolls. Otherwise the sizeForItemAtIndexPath: that we return would be out-of-date.